client/src/views/Dashboard/Summary/index.js

```diff
@@ -423,6 +438,204 @@ export default class Summary extends PureComponent {
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child is Orphan Abandoned Neglected',
+                value: data.totalChildIsOrphanAbandonedNeglected,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child Not in School',
+                value: data.totalChildNotInSchool,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child Living in Public Property Slums',
+                value: data.totalChildLivingInPublicPropertySlums,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child Head of Household',
+                value: data.totalChildHeadOfHousehold,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child Living or Working on the Street',
+                value: data.totalChildLivingOrWorkingOnTheStreet,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child Caregiver',
+                value: data.totalChildCaregiver,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Child living with step parents divorced parents single parent',
+                value: data.totalChildLivingWithStepParentsDivorcedParentsSingleParent,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Extreme Deprivation Total',
+                value: data.totalExtremeDeprivation,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Disaster Prone Area',
+                value: data.totalChildLivingInDisasterProneArea,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Areas With Conflict',
+                value: data.totalChildLivingInAreasWithConflict,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Former War Zones',
+                value: data.totalChildLivingInFormerWarZones,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Affected by Epidemic',
+                value: data.totalChildAffectedByEpidemic,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Catastrophe/  Disaster Total',
+                value: data.totalCatastropheDisaster,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child with Disability',
+                value: data.totalChildWithDisability,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Belongs to a Marginalized Group',
+                value: data.totalChildBelongsToA_marginalizedGroup,
```

Comment:
This doesn't look like a proper camelcase

Reply:
This can be skiped.

Reply:
it's from server